daprdocs/content/en/operations/resiliency/health-checks/sidecar-health.md

```diff
@@ -8,18 +8,47 @@ description: Dapr sidecar health checks
 
 Dapr provides a way to determine its health using an [HTTP `/healthz` endpoint]({{< ref health_api.md >}}). With this endpoint, the *daprd* process, or sidecar, can be:
 
-- Probed for its health
-- Determined for readiness and liveness
+- Probed for its overall health
+- Probed for Dapr sidecar readiness during initialization
+- Determined for readiness and liveness with Kubernetes
 
-In this guide, you learn how the Dapr `/healthz` endpoint integrate with health probes from the application hosting platform (for example, Kubernetes). 
-
-When deploying Dapr to a hosting platform like Kubernetes, the Dapr health endpoint is automatically configured for you.
+In this guide, you learn how the Dapr `/healthz` endpoint integrate with health probes from the application hosting platform (for example, Kubernetes) as well as the Dapr SDKs. 
 
 {{% alert title="Note" color="primary" %}}
 Dapr actors also have a health API endpoint where Dapr probes the application for a response to a signal from Dapr that the actor application is healthy and running. See [actor health API]({{< ref "actors_api.md#health-check" >}}).
 {{% /alert %}} 
 
+The following diagram shows the steps when a Dapr sidecar starts, the healthz endpoint and when the App channel is initialized.  
+
+<img src="/images/healthz-outbound.png" width="800" alt="Diagram of Dapr checking oubound health connections." />
+
+## Outbound health endpoint
+
+As shown by the red boundary lines in the diagram above, the `v1.0/healthz/` endpoint is used to wait for when:
+- All components are initialized;
+- The Dapr HTTP port is available; _and,_
+- The app channel is initialized. 
+
+This is used to check the complete initialization of the Dapr sidecar and its health. 
+
```

Comment:
Let's add the DAPR_HEALTH_TIMEOUT environment variable can be used to set the health time. For examples
"Setting the DAPR_HEALTH_TIMEOUT environment variable let's you control the health timeout which can be important in different environments with higher latency for example"